Data.cpp

```diff
@@ -746,28 +748,28 @@ int Data::synthesizeVowelLf(TlModel *tlModel, LfPulse &lfPulse, int startPos, bo
     // Is a new glottal pulse starting?
     // **************************************************************
 
-    if (i == nextPulsePos)
-    {
-      // Get the pulse amplitude and F0.
+	if (i == nextPulsePos)
```

Comment:
It's annoying but please fix the indentation. The coding style is only two spaces per indentation level.

Reply:
Ok, no pb :)